utils/certs/gen_certificates.sh

```diff
@@ -38,146 +38,210 @@
+
+    echo "Required Agent Certificate Files:
+    ${CERTS}/daosCA.crt
+    ${CERTS}/agent.key
+    ${CERTS}/agent.crt"
 }
 
 function generate_admin_cert () {
-	echo "Generating Admin Certificate"
-	# Generate Private key and set its permissions
-	openssl genrsa -out $CERTS/admin.key 4096
-	chmod 0400 $CERTS/admin.key
-	# Generate a Certificate Signing Request (CRS)
-	openssl req -new -config "$CONFIGS/admin.cnf" -key $CERTS/admin.key \
-		-out admin.csr -batch
-	# Create Certificate from request
-	openssl ca -config "$CONFIGS/ca.cnf" -keyfile $PRIVATE/daosCA.key \
-		-cert $CERTS/daosCA.crt -policy signing_policy \
-		-extensions signing_admin -out $CERTS/admin.crt \
-		-outdir $CERTS -in admin.csr -batch
-
-	echo "Required Admin Certificate Files:
-	$CERTS/daosCA.crt
-	$CERTS/admin.key
-	$CERTS/admin.crt"
+    echo "Generating Admin Certificate"
+    # Generate Private key and set its permissions
+    openssl genrsa -out ${CERTS}/admin.key 4096
+    chmod 0400 ${CERTS}/admin.key
+    # Generate a Certificate Signing Request (CRS)
+    openssl req -new -config "$CONFIGS/admin.cnf" -key ${CERTS}/admin.key \
+        -out ${CA_HOME}/admin.csr -batch
+    # Create Certificate from request
+    openssl ca -config "$CA_HOME/ca.cnf" -keyfile ${PRIVATE}/daosCA.key \
+        -cert ${CERTS}/daosCA.crt -policy signing_policy \
+        -extensions signing_admin -out ${CERTS}/admin.crt \
+        -outdir ${CERTS} -in ${CA_HOME}/admin.csr -batch
+
+    echo "Required Admin Certificate Files:
+    ${CERTS}/daosCA.crt
+    ${CERTS}/admin.key
+    ${CERTS}/admin.crt"
 }
 
 function generate_server_cert () {
-	echo "Generating Server Certificate"
-	# Generate Private key and set its permissions
-	openssl genrsa -out $CERTS/server.key 4096
-	chmod 0400 $CERTS/server.key
-	# Generate a Certificate Signing Request (CRS)
-	openssl req -new -config "$CONFIGS/server.cnf" -key $CERTS/server.key \
-		-out server.csr -batch
-	# Create Certificate from request
-	openssl ca -config "$CONFIGS/ca.cnf" -keyfile $PRIVATE/daosCA.key \
-		-cert $CERTS/daosCA.crt -policy signing_policy \
-		-extensions signing_server -out $CERTS/server.crt \
-		-outdir $CERTS -in server.csr -batch
-
-	echo "Required Server Certificate Files:
-	$CERTS/daosCA.crt
-	$CERTS/server.key
-	$CERTS/server.crt"
+    echo "Generating Server Certificate"
+    # Generate Private key and set its permissions
+    openssl genrsa -out ${CERTS}/server.key 4096
+    chmod 0400 ${CERTS}/server.key
+    # Generate a Certificate Signing Request (CRS)
+    openssl req -new -config "$CONFIGS/server.cnf" -key ${CERTS}/server.key \
+        -out ${CA_HOME}/server.csr -batch
+    # Create Certificate from request
+    openssl ca -config "$CA_HOME/ca.cnf" -keyfile ${PRIVATE}/daosCA.key \
+        -cert ${CERTS}/daosCA.crt -policy signing_policy \
+        -extensions signing_server -out ${CERTS}/server.crt \
+        -outdir ${CERTS} -in ${CA_HOME}/server.csr -batch
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]